lib-database/src/commonMain/kotlin/com/ustadmobile/core/db/dao/ClazzDao.kt

```diff
@@ -66,6 +67,26 @@ abstract class ClazzDao : BaseDao<Clazz> {
     @Insert
     abstract fun insertAuditLog(entity: AuditLog): Long
 
+
+    @Query("SELECT * FROM Clazz WHERE clazzSchoolUid = :schoolUid " +
+            "AND CAST(isClazzActive AS INTEGER) = 1 ")
+    abstract suspend fun findAllClazzesBySchool(schoolUid: Long): List<Clazz>
+
+    @Query("SELECT * FROM Clazz WHERE clazzSchoolUid = :schoolUid " +
+            "AND CAST(isClazzActive AS INTEGER) = 1 ")
+    abstract fun findAllClazzesBySchoolLive(schoolUid: Long)
+            : DataSource.Factory<Int,Clazz>
+
+    @Query("UPDATE Clazz SET isClazzActive = :active WHERE clazzUid = :clazzUid ")
+    abstract suspend fun updateActiveByClazzUid(clazzUid: Long, active : Boolean)
+
+    override suspend fun deactivateByUids(uidList: List<Long>) {
```

Comment:
If someone deactivates a class from the school page, I think this should just break the connection between it and the school, not deactivate the whole class. We also wouldn't want to do that without some kind of double confirmation (e.g. dialog). I think this should set clazzSchoolUid = 0 instead of setting active to false.